actors/builtin/power/power_actor.go

```diff
@@ -33,8 +33,8 @@ func (a Actor) Exports() []interface{} {
 		4:                         a.EnrollCronEvent,
 		5:                         a.OnEpochTickEnd,
 		6:                         a.UpdatePledgeTotal,
-		7:                         a.SubmitPoRepForBulkVerify,
-		8:                         a.CurrentTotalPower,
+		8:                         a.SubmitPoRepForBulkVerify,
+		9:                         a.CurrentTotalPower,
```

Comment:
Please add the placeholder for 7 so the gap is obvious.

What was method 7?

Reply:
Yep. Done.